actioncable/app/assets/javascripts/action_cable/subscription.coffee

```diff
@@ -63,6 +63,9 @@ class ActionCable.Subscription
     @consumer.send(command: "message", identifier: @identifier, data: JSON.stringify(data))
 
   unsubscribe: ->
+    @consumer.send(command: "unsubscribe", identifier: @identifier)
+
+  unsubscribed: ->
```

Comment:
This is the method that user-created subscriptions are intended to implement to be notified of `unsubscribed`. Meaning `@subscriptions.notify(identifier, "unsubscribed")` calls this so we can't have our own method with the same name.